Core/WebViewController.swift

```diff
@@ -47,7 +51,12 @@ open class WebViewController: UIViewController {
     @IBOutlet weak var webViewContainer: UIView!
 
     open private(set) var webView: WKWebView!
-
+    
+    /// Override to change the desktop mode
+    open var isDesktopMode: Bool {
```

Comment:
Optional: I know that in the long term that we will merge the inheritance here but I'm still uncomfortable with the default property that is essentially useless until it is overridden by a subclass. Does it make sense to make the desktop mode a concern of the `TabViewController`? You could add delegate methods to let `TabViewController` know when load and reload happen and it can then own and call `updateUserAgent`.

Reply:
Have made "userAgent" a property of the web view controller rather than "is desktop" and the tab view controller is now responsible for setting that as appropriate.